### ISSUE TITLE
Add support for full root encryption (chrx with Gallium)

### ISSUE THREAD

**Issue by dayschmer** [reynhout/chrx#58]
A feature request: Would it be possible to add full root encryption for chrx with Gallium? At the moment I have my home directory encrypted, but it'd be nice to encrypt the whole file system.

Thanks for all of your work on chrx and Gallium. It's wonderful to have a full Linux distribution working on a Chromebook. And Gallium not only works but looks great too.

**Comment by mrjohnc** [comment]
Yes please, this would be really nice to have, either/both during installation or a clear guide on how to do this after installation
